next/components/molecules/SearchPageNew/searchDataFetchers.ts

```diff
@@ -181,10 +180,9 @@ const getSearchUsersData = (filters: PagesFilters): SearchResponse => {
     data?.data.map((user) => {
       return {
         title: user.displayName,
-        metadata: [user.jobTitle],
+        metadata: [user.jobTitle, user.mail, user.businessPhones?.join(', ')],
```

Comment:
Please check what mails and phones are used in `OrganizationalStructureAccordionCard.tsx`.
E.g. primator shoulh have mail primator@bratislava.sk - it's specified as `otherEmails[0]` (several more people have email like this - director etc.).

(Do not reuse mail parsing and splitting, that's legacy)

![image](https://github.com/bratislava/bratislava.sk/assets/19418224/716cea87-1667-42f0-97f4-da922b0bb98d)
![image](https://github.com/bratislava/bratislava.sk/assets/19418224/30beb9b9-41e3-48ef-a646-341286b61167)